<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sheetsavvy_main_window</class>
 <widget class="QMainWindow" name="sheetsavvy_main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SheetSavvy</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="200,100">
      <item row="0" column="1">
       <widget class="QLabel" name="save_file_image_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="reset_button">
          <property name="text">
           <string>Retry Connection</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save As</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QListView" name="device_info_list"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>552</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
